SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -38,10 +38,10 @@ You can also access these from the **Window** menu:
 Generates a [SpatialOS snapshot](https://docs.improbable.io/reference/latest/shared/glossary#snapshot) (SpatialOS documentation).
 
 * Codegen</br>
-Creates [SpatialOS Unreal GDK interop code]({{urlRoot}}/content/interop).
+Creates `schema` for your Unreal project.
 
 * Launch</br>
-Runs `spatial local launch` with the launch configuration specified in the settings (see [below](#settings)).
+Builds `worker configs` and runs `spatial local launch` with the launch configuration specified in the settings (see [below](#settings)).
```

Comment:
runs `spatial worker build build-config` to build worker configurations <link>

Reply:
I hate you so much